src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const transponsedRobot = {};
```

Comment:
There is a small typo in the variable name, it should be transposedRobot instead of transponsedRobot